packages/core/src/components/button/_common.scss

```diff
@@ -184,6 +188,10 @@ $non-default-state-selectors: ":last-child, :hover, :active, .pt-active, [class*
     background-image: none;
     color: $dark-button-color-disabled;
   }
+
+  .pt-button-spinner-container .pt-spinner-head {
```

Comment:
Well we don't know what content the user puts in the `Button` right, so they might put in a `Spinner intent={Intent.Primary}` and be sad that the stroke isn't what they expect?

Perhaps they shouldn't be doing that, but I'd argue that its better to just faithfully render `children` without overriding its styles